src/Interpreters/InterpreterSelectQuery.cpp

```diff
@@ -1645,7 +1645,6 @@ void InterpreterSelectQuery::executeImpl(QueryPlan & query_plan, std::optional<P
                         expressions.join,
                         settings.max_block_size);
 
-                    filled_join_step->setStepDescription("JOIN");
```

Comment:
Why?

Reply:
Incorrect merge with https://github.com/ClickHouse/ClickHouse/pull/53006, will restore this line